File: patterns/java/src/refactoring_guru/patterns/builder/example/cars/Manual.java
package refactoring.guru.patterns.builder.example.cars;

import refactoring.guru.patterns.builder.example.components.Engine;
import refactoring.guru.patterns.builder.example.components.GPSNavigator;
import refactoring.guru.patterns.builder.example.components.Transmission;
import refactoring.guru.patterns.builder.example.components.TripComputer;

public class Manual {
    private final Type type;
    private final int seats;
    private final Engine engine;
    private final Transmission transmission;
    private final TripComputer tripComputer;
    private final GPSNavigator gpsNavigator;

    public Manual(Type type, int seats, Engine engine, Transmission transmission,
               TripComputer tripComputer, GPSNavigator gpsNavigator) {
        this.type = type;
        this.seats = seats;
        this.engine = engine;
        this.transmission = transmission;
        this.tripComputer = tripComputer;
        this.gpsNavigator = gpsNavigator;
    }

    public String print() {
        StringBuilder info = new StringBuilder();
        info.append("Type of car: " + type + "\n");
        info.append("Count of seats: " + seats + "\n");
        info.append("Engine: volume - " + engine.getVolume() + "; mileage - " + engine.getMileage() + "\n");
        info.append("Transmission: " + transmission + "\n");
        info.append("Trip Computer: Trip Computer" + "\n");
        info.append("GPS Navigator: GPS Navigator" + "\n");
        return info.toString();
    }
}

File: patterns/cs/src/RefactoringGuru/Builder/Example/Cars/Manual.cs
using System.Text;
using RefactoringGuru.Builder.Example.Components;

namespace RefactoringGuru.Builder.Example.Cars
{
    /**
     * EN: Car manual is another product. Note that it does not have the same
     * ancestor as a Car. They are not related.
     * 
     * RU: Руководство автомобиля — это второй продукт. Заметьте, что руководство и
     * сам автомобиль не имеют общего родительского класса. По сути, они независимы.
     */
    public class Manual
    {
        private Cars.Type type;
        private int seats;
        private Engine engine;
        private Transmission transmission;
        private TripComputer tripComputer;
        private GPSNavigator gpsNavigator;

        public Manual(Type type, int seats, Engine engine, Transmission transmission, TripComputer tripComputer, GPSNavigator gpsNavigator)
        {
            this.type = type;
            this.seats = seats;
            this.engine = engine;
            this.transmission = transmission;
            this.tripComputer = tripComputer;
            this.gpsNavigator = gpsNavigator;
        }

        public virtual string Print()
        {
            StringBuilder info = new StringBuilder();
            info.Append("Type of car: " + type + "\n");
            info.Append("Count of seats: " + seats + "\n");
            info.Append("Engine: volume - " + engine.GetVolume() + "; mileage - " + engine.GetMileage() + "\n");
            info.Append("Transmission: " + transmission + "\n");
            info.Append("Trip Computer: Trip Computer" + "\n");
            info.Append("GPS Navigator: GPS Navigator" + "\n");
            return info.ToString();
        }
    }
}
